src/main/java/org/broadinstitute/consent/http/service/MetricsService.java

```diff
@@ -53,7 +53,7 @@ public MetricsService(DacService dacService, DatasetDAO dataSetDAO, DataAccessRe
   }
 
   public class DarMetricsSummary {
-    @JsonProperty final Timestamp updateDate;
+    final Timestamp updateDate;
     @JsonProperty final String projectTitle;
```

Comment:
In a follow on story, we should see if we can completely rip out all `@JsonProperty` annotations. This is a jackson thing and if we aren't using it, we should be able to excise it from dependencies too.

Reply:
Agreed - I started to do that in this ticket, but then quickly realized it was probably out of the scope.